packages/website/components/SidebarSection.tsx

```diff
@@ -98,6 +98,16 @@ export function SidebarSection({
   const { currentPage } = useCurrentLocation();
   const { data: session } = useSession();
 
+  // move the deprecated link section to the end of the components list
+  const deprecatedLinkSectionIndex = links.findIndex((link) =>
```

Comment:
I am a bit ignorant about how the repo is setup but this filtering on the links feels like business logic that the `SidebarSection` should be agnostic about.

Where are the `links` being passed down as props? Can we do the filtering there and just pass down the links properly filtered? Also can we extract this logic into a function with a fitting name (e.g. `sortLinks`?) so that it's easier to read.

Reply:
Well, you actually raise a good point, I could actually manually place it in the code below, no need to manipulate the links.

https://github.com/contentful/forma-36/blob/af819f3fe8be9f4ea012544b82feb8cbd99c9731/packages/website/pages/%5B...slug%5D.tsx#L166-L171

The only difference will be that it will not be a collapsable section, but its own expanded section, which seems fine to me.

<img width="291" alt="Screenshot 2023-07-21 at 17 18 51" src="https://github.com/contentful/forma-36/assets/103024358/ac342dd5-fa52-4e5f-ac57-f57ce808d125">

Reply:
No strong opinions here as I'm lacking context, the others can plus 1 it, but it sounds fine to me.

Reply:
@mshaaban0 what do you reckon on this one ☝️ ?